test/modules/aggregation-layers/contour-layer/marching-squares.spec.js

```diff
@@ -393,6 +450,33 @@ const GETVERTEX_TESTS = [
   }
 ];
 
```

Comment:
Did you add any new tests similar to `GETVERTEX_TESTS` that operate using computed threshold. The key of this feature is how vertices are adjusted based on neighbor cell weights, we should have these unit tests for different cases, and validate functionality, otherwise it is hard to find these problem looking at rendered images.